docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID
+---
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived certificates issued to tbot users on Teleport clusters can be successfully applied.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+- A host that you wish to assign an identity to using Machine ID.
+
+## Step 1/4. Download and install Teleport (=teleport.version=)
+
+In this step, you will be downloading and installing Teleport binaries onto the
+machine you wish to assign an identity to.
+
+Each Teleport package hosted on our downloads page ships with several useful
+binaries, including `teleport`, `tctl`, `tsh`, and `tbot`:
+
+- `teleport` is the daemon used to initialize a Teleport cluster; this binary is not used in this guide.
+- `tctl` is the administrative tool you will use to create the bot user.
+- `tsh` is the client tool you will use to log in to the Teleport Cluster.
+- `tbot` is the Machine ID tool you will use to associate a bot user with a machine.
+
+Next, log in to your cluster using `tsh` to ensure that your machine will be able to interact with the cluster:
```

Comment:
This step says to download and install teleport, but it doesn't tell you where to download it from or how to install it - it skips straight to logging in.

Reply:
My mistake here, adding a link to the [downloads page](https://goteleport.com/teleport/download) and a little extra background info